guide/src/advanced/features.md

```diff
@@ -110,6 +110,8 @@ __Feature comparison between Hermes and the Go relayer__
 |                        |       |        |
 | FT_Transfer            | ✅    | ✅     | can submit an ICS-20 fungible token transfer message
 | ICA_Relay               | ✅    | ✅     | can relay ICS-27 Interchain account packets
+| Interchain Query (ICQ) support |   ✅    |     ✅     | interchain querying using ABCI
+| Cross-chain Queries            |   ❌    |     ✅     | cross-chain querying between IBC-enabled chains
```

Comment:
```suggestion
| Cross-chain Queries            |    ✅    |     ✅     | cross-chain querying between IBC-enabled chains
```
Hermes is compatible with both ICS31 and async ICQs:
* https://github.com/informalsystems/hermes/actions/runs/7058621945/job/19214580020?pr=3711#step:9:29
* https://github.com/informalsystems/hermes/actions/runs/7058621945/job/19214577684?pr=3711#step:9:28